acme-cli/build.gradle

```diff
@@ -69,3 +65,27 @@ shadowJar {
 
 run.classpath += configurations.developmentOnly
 run.jvmArgs('-noverify', '-XX:TieredStopAtLevel=1', '-Dcom.sun.management.jmxremote')
+
+shadowJar {
+    archiveBaseName = "micronaut-" + project.name.substring(project.name.indexOf('/') + 1)
```

Comment:
Think you can get rid of the all the substrings now

Reply:
Fixed here 2c647f7